core-bundle/src/DataContainer/ButtonsBuilder.php

```diff
@@ -102,7 +103,7 @@ public function generateSelectButtons(string $strTable, bool $isMovable, DataCon
             $arrButtons['cut'] = '<button type="submit" name="cut" id="cut" class="tl_submit" accesskey="x">'.$GLOBALS['TL_LANG']['MSC']['moveSelected'].'</button>';
         }
 
-        if (!($GLOBALS['TL_DCA'][$strTable]['config']['notEditable'] ?? null)) {
+        if (!($GLOBALS['TL_DCA'][$strTable]['config']['notEditable'] ?? null) && !$dc instanceof DC_Folder) {
```

Comment:
That looks wrong to me. What if I have my own `DC_Foobar` that does also not support it? /cc @aschempp

Reply:
Maybe you can fix it by defining the operations manually in `tl_files`?